src/main/java/org/sagebionetworks/web/client/widget/entity/EntityViewScopeEditorModalWidget.java

```diff
@@ -0,0 +1,70 @@
+package org.sagebionetworks.web.client.widget.entity;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+import org.sagebionetworks.repo.model.Reference;
+import org.sagebionetworks.web.client.GlobalApplicationState;
+import org.sagebionetworks.web.client.jsinterop.EntityViewScopeEditorModalProps;
+import org.sagebionetworks.web.client.jsinterop.SqlDefinedTableEditorModalProps;
+import org.sagebionetworks.web.client.widget.table.modal.fileview.TableType;
+
+public class EntityViewScopeEditorModalWidget implements IsWidget {
+
+  private final GlobalApplicationState globalApplicationState;
+  private final EntityViewScopeEditorModalWidgetView view;
+
+  private String entityId;
+  private EntityViewScopeEditorModalProps.Callback onCancel;
+  private EntityViewScopeEditorModalProps.Callback onUpdate;
+  boolean open;
+
+  @Inject
+  public EntityViewScopeEditorModalWidget(
+    EntityViewScopeEditorModalWidgetView view,
+    GlobalApplicationState globalApplicationState
+  ) {
+    super();
+    this.view = view;
+    this.globalApplicationState = globalApplicationState;
+  }
+
+  public void configure(
+    String entityId,
+    EntityViewScopeEditorModalProps.Callback onUpdate,
+    EntityViewScopeEditorModalProps.Callback onCancel,
+    boolean open
```

Comment:
for consistency with SqlDefinedEditorModalWidget.java, recommend removing `open` from the `configure` signature and making the initial value of the prop `false`.